GeometryTools/Rectangle.swift

```diff
@@ -190,15 +190,25 @@ public struct Rectangle: ConvexPolygonProtocol {
     /// - Returns: `Rectangle` with dimensions scaled by the given `value` around the given
     /// `anchor`.
     public func scaled(by value: Double, around anchor: ScaleAnchor) -> Rectangle {
+        return scaledBy(width: value, height: value, around: anchor)
+    }
+
+    /// - Returns: `Rectangle` with dimensions scaled by the given `width` and `height`, around
+    /// the given `anchor`.
+    public func scaledBy(
+        width widthScale: Double = 1,
+        height heightScale: Double = 1,
+        around anchor: ScaleAnchor
+        ) -> Rectangle
+    {
+        let newSize = self.size.scaledBy(width: widthScale, height: heightScale)
```

Comment:
`self` is only required if disambiguating a local variable and an instance variable.

I would say either `let size = self.size.scaledBy(...)` or `let newSize = size.scaledBy(...)`